src/pages/index.js

```diff
@@ -153,7 +153,7 @@ function handleAvatarChange(data) {
       avatarPopup.renderLoading(false)
```

Comment:
` avatarPopup.renderLoading(false)` should be in `finally`


You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.